DAO.sol

```diff
@@ -902,16 +954,28 @@ contract DAO is DAOInterface, Token, TokenCreation {
         if (blocked[_account] == 0)
             return false;
         Proposal p = proposals[blocked[_account]];
-        if (now > p.votingDeadline) {
+        if (p.open) {
             blocked[_account] = 0;
             return false;
         } else {
             return true;
         }
     }
 
+    function isSplitBlocked(address _account) internal returns (bool) {
```

Comment:
Please change the name of this function here and wherever it is used. The function may also change the `splitBlocked[]` array. So a better name would be: `getOrModifySplitBlocked()`